### DISCUSSION TITLE
[blocks]: landing page

### DISCUSSION THREAD

**Discussion by SepehrShapouri** [shadcn-ui/ui#4348] [Blocks Request]
### Description

A saas landing page would be awesome

**Comment by ObservedObserver** [comment]
Hi @SepehrShapouri, I am working on the landing page blocks generator, `convertfast-ui`, based on shadcn-ui, it provides a CLI tool that you can create beautiful landing pages (based on shadcn-ui) with one command like this:

```bash
npx convertfast@latest page create <your-page-name>
```

It is also open sourced, if you are interested to have a try, check [the github repo](https://github.com/ObservedObserver/convertfast-ui). Or [examples and docs](https://ui.convertfa.st)

Landing page demo:

<img width="1801" alt="convertfast-ui-light-demo" src="https://github.com/user-attachments/assets/6f25d8ab-13dd-42cc-a4b1-88dc8aabe983">
